src/utils/metadata.py

```diff
@@ -9,10 +9,19 @@
 
 
 def _read_in_metadata(metadata, user_type):
+    sounds_dir = get_sounds_dir(user_type)
     for sound_event_id in metadata:
-        metadata[sound_event_id]["sound-file"] =\
-            os.path.join(get_sounds_dir(user_type),
-                         metadata[sound_event_id]["sound-file"])
+        if "sound-files" not in metadata[sound_event_id]:
+            sound_files = []
+        else:
+            sound_files = metadata[sound_event_id]["sound-files"]
```

Comment:
Drive-by review: Instead of the if/else, you can also do: `metadata[sound_event_id].get("sound-files", [])`